network-api/networkapi/mozfest/templates/partials/signup.html

```diff
@@ -11,8 +11,8 @@
           class="join-us mb-5"
           data-form-position="newsletter_signup_sidebar"
           data-signup-id="{{ page.signup.id|unlocalize }}"
```

Comment:
Should the signup ids be the same between the locales? Maybe a question for @TheoChevalier

Reply:
Hey Tibor as far as I know I believe they should be, but maybe @TheoChevalier can confirm

Reply:
Hmm… Sorry, it’s not super clear to me what these IDs are, unless these are the Salesforce IDs? In that case, yes, they should be the same

Reply:
Thanks Theo! After doing some digging, it does look like these are in fact salesforce IDs. 

For reference:
https://github.com/mozilla/foundation.mozilla.org/issues/5712 Here is the issue asking that salesforce IDs be implemented
https://github.com/mozilla/foundation.mozilla.org/pull/5714 Here is the PR adding the ID field 👍 

Looks like they should be the same! @tbrlpld

Reply:
@danielfmiranda I think the `signup.id` is different from the Salesforce ID. If you look at the field name that is added in the PR you linked to, it's `campaign_id` that seems to be the Salesforce ID: 

https://github.com/mozilla/foundation.mozilla.org/pull/5714/files#diff-62de065a3de1e8fe56917af0194a4239e0575943f3b8ecac0235a496db99eb2cR57